components/text_normalization/fondant_component.yaml

```diff
@@ -0,0 +1,20 @@
+name: Normalize text.
+description: A component that normalizes text.
+image: ghcr.io/ml6team/text_normalization:latest
```

Comment:
```suggestion
image: ghcr.io/ml6team/normalize_text:latest
```

Reply:
As mentioned in chat, agree with consistent naming, but would actually prefer `text_normalization` here as it will group all text components alphabetically.

Reply:
Hmm not necessarily agree on that one, we can group them ourselves in the docs?

Would prefer to have verb + noun for all our components